src/r/build_package.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -ex
+
+# Creates the PsychroLib package repository to deploy locally.
+# Usage: build_package.sh
+
+if [ ! -d R ]; then
+    echo "Please run this script from r project folder"
+    exit 1
+fi
+
+R -e 'devtools::document()'
+cp ../../LICENSE.txt .
+R -e 'devtools::install()'
+rm -rf man
```

Comment:
Why are you removing the `man` folder

Reply:
This is because `devtools::install()` will compile the package source code, including files the `man` folder and install them into the default library location. After that, the `man` folder is not needed anymore.

Reply:
see https://github.com/psychrometrics/psychrolib/pull/49#discussion_r368564328